sporkfish/searcher/minimax.py

```diff
@@ -77,6 +79,13 @@ def __init__(
             == MoveOrderMode.COMPOSITE
             else None
         )
+        if (
+            self._searcher_config.move_order_config.move_order_mode
+            == MoveOrderMode.HISTORY
+            or self._searcher_config.move_order_config.move_order_mode
+            == MoveOrderMode.COMPOSITE
+        ):
+            self._history_table: Dict[chess.Move, int] = dict()
```

Comment:
I suggest adding a comment that this is a slight deviation to the original description of history heuristic on chess programming wiki - basically by using chess.Move we are including (from,to,promotion) instead of just (from,to).

Reply:
Do you guys think it is better to add the comment maybe in history_heuristic.py?

Reply:
OK, sounds good.